Mathlib/Combinatorics/Enumerative/Bell.lean

```diff
@@ -168,4 +175,33 @@ theorem uniformBell_eq_div (m : ℕ) {n : ℕ} (hn : n ≠ 0) :
   · exact Nat.mul_pos (Nat.pow_pos (Nat.factorial_pos n)) m.factorial_pos
   · rw [← mul_assoc, ← uniformBell_mul_eq _ hn]
 
+/-- The `n`th standard Bell number,
+which counts the number of partitions of a set of cardinality `n`. -/
+def standardBell : ℕ → ℕ
+  | 0 => 1
+  | n + 1 => ∑ k ∈ Finset.range (n + 1), Nat.choose n k * standardBell (n - k)
```

Comment:
I think this might be better summing over `Finset.antidiagonal`, which would eliminate the `-`. Let's see what other reviewers think.

Reply:
Indeed. And one can always rewrite the `Finset.antidiagonal` sum if needed, thanks to docs#Finset.Nat.sum_antidiagonal_eq_sum_range_succ